oci/layout/oci_transport_test.go

```diff
@@ -23,6 +23,18 @@ func TestGetManifestDescriptor(t *testing.T) {
 
 	_, err = imageRef.(ociReference).getManifestDescriptor()
 	assert.EqualError(t, err, ErrMoreThanOneImage.Error())
+
```

Comment:
The comment above the function now needs to be moved to the appropriate test case.